tidb-cloud/data-service-response-and-status-code.md

```diff
@@ -27,37 +27,34 @@ An example response is as follows:
 
 ```json
 {
-  "type": "sql_endpoint",
-  "data": {
-    "columns": [
-      {
-        "col": "id",
-        "data_type": "BIGINT",
-        "nullable": false
-      },
-      {
-        "col": "type",
-        "data_type": "VARCHAR",
-        "nullable": false
-      }
-    ],
-    "rows": [
-      {
-        "id": "20008295419",
-        "type": "CreateEvent"
-      }
-    ],
-    "result": {
-      "code": 200,
-      "message": "Query OK!",
-      "start_ms": 1678965476709,
-      "end_ms": 1678965476839,
-      "latency": "130ms",
-      "row_count": 1,
-      "row_affect": 0,
-      "limit": 50
+    "type": "sql_endpoint",
+    "data": {
+        "columns": [],
+        "rows": [
+            {
+                "auto_increment_id": "270001",
```

Comment:
I think auto_increment_id field is only populated for an INSERT operation that too only when the primary key column is configured as auto_increment  in the table. 

This is not mentioned in the documentation. Can you please confirm with Hongmeng and add it to the documentation please?

Reply:
According to Hongmeng, auto_increment_id will be added for response of all batch post/put/delete requests when the primary key column is configured as auto_increment in the table.

Reply:
updated in https://github.com/pingcap/docs/pull/14244/commits/670cef6ebb7889a41fcd3676a1d7cd78f47df680